sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -505,7 +505,13 @@ class PlanParserSuite extends PlanTest {
     val m2 = intercept[ParseException] {
       parsePlan("SELECT /*+ MAPJOIN(default.t) */ * from default.t")
     }.getMessage
-    assert(m2.contains("no viable alternative at input"))
+    assert(m2.contains("mismatched input '.' expecting {')', ','}"))
+
+    // Disallow space as the delimiter.
+    val m3 = intercept[ParseException] {
```

Comment:
NIT: You can use the `PlanParserSuite.intercept` method. That saves some typing.